etc/profile-m-z/vmware.profile

```diff
@@ -0,0 +1,41 @@
+# Firejail profile for vmware
+# Description: The industry standard for running multiple operating systems as virtual machines on a single Linux PC.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include vmware.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.vmware
+noblacklist ${HOME}/.cache/vmware
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.vmware
+mkdir ${HOME}/.cache/vmware
+whitelist ${HOME}/.vmware
+whitelist ${HOME}/.cache/vmware
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+caps.keep sys_nice
+ipc-namespace
+netfilter
+
+
```

Comment:
GitHub suggestions are still not perfect. Remove the blank lines.

Reply:
done